src/EnergyPlus/Pumps.cc

```diff
@@ -1816,8 +1816,8 @@ void CalcPumps(EnergyPlusData &state, int const PumpNum, Real64 const FlowReques
     // Get RPM value for reporting as output
     // RPM is calculated using pump affinity laws for rotation speed
     if (thisPumpPlant.UsePressureForPumpCalcs && thisPump.HasVFD) {
-        RotSpeed_Min = thisPump.VFD.minRPMSched->getCurrentVal();
-        RotSpeed_Max = thisPump.VFD.maxRPMSched->getCurrentVal();
+        RotSpeed_Min = thisPump.VFD.minRPMSched ? thisPump.VFD.minRPMSched->getCurrentVal() : 0.0;
+        RotSpeed_Max = thisPump.VFD.maxRPMSched ? thisPump.VFD.maxRPMSched->getCurrentVal() : 0.0;
```

Comment:
I'm surprised this would crash. If HasVFD = true then min/maxRPMSched must be valid schedules.